src/garage/torch/algos/ppo.py

```diff
@@ -59,7 +61,9 @@ def __init__(self,
                  policy_ent_coeff=0.0,
                  use_softplus_entropy=False,
                  stop_entropy_gradient=False,
-                 entropy_method='no_entropy'):
+                 entropy_method='no_entropy',
+                 minibatch_size=None,
+                 max_optimization_epochs=1):
```

Comment:
please give this a sane default. the default configuration of PPO should use a minibatching to maximum effect.

Reply:
fixed